api/services/ocean/spark/parameters/query/maxCreatedAt.yaml

```diff
@@ -0,0 +1,7 @@
+in: query
+name: maxCreatedAt
+required: false
+description: Optionally filter on the creation date
```

Comment:
For all the date-time fields, can we give an example ?

Right now the HTML just says "date-time", while if you said "YYYY-MM-DD" it would be more helpful.
<img width="395" alt="Screenshot 2021-11-22 at 13 26 10" src="https://user-images.githubusercontent.com/8062861/142861743-5797c91d-a37c-4537-bc0c-f9f0519230c4.png">

Reply:
Done!